angular-portfolio/src/app/awards/awards.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="awards">
+    <mat-toolbar color="primary">
+        <span>Awards and Honors</span>
+    </mat-toolbar>
+    <div class="flex-container">
+        <div class="years">
+            <div *ngFor="let awd of awards; let i=index"
+            (click)="onSelect(awd)">
+                <ng-container *ngIf="i === 0 || awd.year !== awards[i-1].year">
+                    <button mat-raised-button color="primary">{{awd.year}}</button>
```

Comment:
Does this button do anything?

Reply:
Technically, no. I use the onSelect() function to look at the element, but I realize that element could be anything and there is nothing special about the button as I do not look at the button's value. I only used the button as I liked the angular material design; however, I understand this is bad practice.